src/functions/sendslack/timebankdata.ts

```diff
@@ -0,0 +1,84 @@
+const fetch = require('node-fetch');
+import { PersonDto, TimeEntryTotalDto } from "./schema";
+
+/**
+ * 
+ * @returns Working hours and Persons information
+ */
+
+const timebankData = async () => {
```

Comment:
This file should have custom namespace (something like TimebankApiProvider) and all functions that are needed outside this file should be defined as export const ...